chart/templates/_helpers.tpl

```diff
@@ -345,3 +345,17 @@ clientCache numLocks
 {{- end -}}
 {{- end -}}
 {{- end -}}
+
+{{/*
+topologySpreadConstraints appends the "vso.chart.selectorLabels" to .Values.controller.topologySpreadConstraints if no labelSelector was specified
```

Comment:
Since we're also adding topology spread constaints, not just priority class and pod disruption budget, can you modify the PR name so that it's included as well? Just so that we don't forget to mention this field when we make the changelog, as some users may be excited to learn about it. (assuming the PR name is what gets used for the changelog entries)